src/main/java/org/jvnet/hudson/test/junit/jupiter/JenkinsRule.java

```diff
@@ -0,0 +1,57 @@
+package org.jvnet.hudson.test.junit.jupiter;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * JUnit 5 meta annotation providing {@link org.jvnet.hudson.test.JenkinsRule JenkinsRule} integration.
+ *
+ * <p>
+ * Test methods using the rule extension need to accept it by {@link JenkinsRuleExtension} parameter; each test case
+ * gets a new rule object.
+ * <p>
+ * Annotating a <i>class</i> provides access for all of it's tests. Unrelated test cases can omit the parameter.
+ *
+ * <blockquote><pre>
+ * &#64;JenkinsRule
```

Comment:
Does `@code` suffice to let you avoid escaping `@`?

Reply:
As explained in https://reflectoring.io/howto-format-code-snippets-in-javadoc/#pre--code , with @code, `@` are interpreted as javadoc tags